### PR TITLE
feat(groups): allow group invitations to be resend

### PR DESCRIPTION
**Pull request by jeabakker**
fixes: #4363

### CODE REVIEW

File: mod/groups/actions/groups/membership/invite.php [hunk 1]
```diff
@@ -22,23 +22,32 @@
...
 		continue;
 	}
 
+	$invite = true;
 	if (check_entity_relationship($group->guid, 'invited', $user->guid)) {
-		register_error(elgg_echo('groups:useralreadyinvited'));
-		continue;
+		// user already invited, do we need to resend the invitation
+		if (!$resend_invitation) {
+			register_error(elgg_echo('groups:useralreadyinvited'));
```
Comment (jdalsem) [inline mod/groups/actions/groups/membership/invite.php]: maybe add the name of the user to the message

**Review by jdalsem — APPROVED** [review]

**Comment by jdalsem** [comment]
@mrclay should the commit message say 'resended'? English grammer...

**Comment by hypeJunction** [comment]
Gramm**a**tically correct past form of send is sent, hence **resent**

**Comment by jdalsem** [comment]
So it is wrong :) as commit message uses **resend**

**Pull request merged**